### ISSUE TITLE
Playing one OGG file after another appears to cause noise...

### ISSUE THREAD

**Issue by Sslaxx** [godotengine/godot#13084]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Ubuntu 17.10 64-bit, 3.0 build 341bf71.

**Issue description:**
Appears to be related to #13058 and (indirectly) #9793.

I have a piece of looping music (OGG format) that plays, until game over, at which point it is replaced by a non-looping piece of music (OGG format).

The non-looping music plays, but once it finishes playing it appears to be making a continuous noise (the last sample of the OGG file?) until restart.

It also spams on the command line:
```
ERROR: _mix_internal: Condition ' !active ' is true.
   At: modules/stb_vorbis/audio_stream_ogg_vorbis.cpp:45.
```

**Steps to reproduce:**
Apparently, have an AudioStreamPlayer set to initially play a looping piece of music, then replace with a non-looping piece.

**Link to minimal example project:**
https://www.dropbox.com/s/xe1zbl0zjoodg6q/13084_demonstrator.webm?dl=0 is a video that demonstrates the issue. https://www.dropbox.com/s/w4kymuayrwvek9o/13084_demonstrator.7z?dl=0 is the source code.

**Comment by Sslaxx** [comment]
Can confirm it is an issue with the end sample(s) of the OGG file; adding a silence of 0.10 at the end eliminates the issue.

**Comment by Sslaxx** [comment]
Fixed by 139daf0 and 045f810.
